README.md

```diff
@@ -53,66 +79,93 @@ Point tags are key-value pairs (strings) that are associated with a point. Point
+	return "Hello World", nil
+}
+
+func main() {
+	// Wrap the handler with wfAgent.WrapHandler()
+	lambda.Start(wfAgent.WrapHandler(handler))
+}
+```
+
+## Metrics
+
+### Standard Metrics
+
+The Wavefront Agent will send a set of default metrics to Wavefront when enabled. The metrics reported are:
 
 | Metric Name                       |  Type         | Description                                                             |
 | --------------------------------- | ------------- | ----------------------------------------------------------------------- |
-| aws.lambda.wf.invocations.count   | Delta Counter | Count of number of lambda function invocations aggregated at the server.|
+| aws.lambda.wf.invocations.count   | Delta Counter | Count of number of Lambda function invocations aggregated at the server.|
 | aws.lambda.wf.errors.count        | Delta Counter | Count of number of errors aggregated at the server.                     |
 | aws.lambda.wf.coldstarts.count    | Delta Counter | Count of number of cold starts aggregated at the server.                |
-| aws.lambda.wf.duration.value      | Gauge         | Execution time of the Lambda handler function in milliseconds.          |
-
-## Custom Metrics
+| aws.lambda.wf.duration.value      | Metric        | Execution time of the Lambda handler function in milliseconds.          |
+| aws.lambda.wf.mem.total           | Metric        | The total memory available to the Lambda function in megabytes.         |
+| aws.lambda.wf.mem.used            | Metric        | The memory used by the Lambda function in megabytes.                    |
+| aws.lambda.wf.mem.percentage      | Metric        | The percentage of memory used by the Lambda function.                   |
 
-You can send custom business metrics to Wavefront using the [go-metrics-wavefront](https://github.com/wavefrontHQ/go-metrics-wavefront) client. The below code reports a _counter_, a _delta counter_, and two _gauges_. All metric names should be unique. If you have metrics that you want to track as both _counter_ and _delta counter_, you'll have to add a suffix to one of the metrics. Having the same metric name for any two types of metrics will result in only one time series at the server and thus cause collisions.
+### Custom Metrics
 
-The code below imported this module and wrapped the _handler_ function argument in _main_ with `wflambda.Wrapper(handler)`. During each execution four metrics are collected and sent to Wavefront with both the [standard point tags](#standard-point-tags) and the point tags created in the handler.
+You can send custom business metrics to Wavefront using the `RegisterMetric()` or `RegisterCounter()` methods. Counters are values that are aggregated at the Wavefront server (like the number of invocations and metrics are pretty much every other numerical value you want to send in.
```

Comment:
It would be great to have a  "things to modify for upgrade" section which details things required to change in the client code, if user is upgrading to the new wavefront-lambda-go library.

Reply:
I agree, I've added that as a task to be completed once we agree the code is ready.